site/dat/repo/various.json

```diff
@@ -2099,5 +2099,47 @@
         "downloadUrl": "https://github.com/Qytera-Gmbh/JMeterHARImporterPlugin/releases/download/v0.2.3/jmeter-har-import-plugin-v0.2.3.jar"
       }
     }
+  },
+  {
+    "id": "JmeterNamesTemplater",
+    "name": "JMeter Names Templater",
+    "description": "Jmeter Names Templater is a plugin for Jmeter that allows you to set naming templates for elements and rename them based on a described configuration file.",
+    "vendor": "Daniil Golikov",
+    "markerClass": "dg.jmeter.plugins.templater.gui.RenameElementsMenuCreator",
+    "helpUrl": "https://github.com/DanilGolikov/JmeterNamesTemplater/blob/master/README.md",
+    "componentClasses": [
+      "dg.jmeter.plugins.templater.gui.RenameElementsMenuCreator",
+      "dg.jmeter.plugins.templater.gui.PrintTreeItem",
+      "dg.jmeter.plugins.templater.gui.RenameSelectedTreeItem",
+      "dg.jmeter.plugins.templater.gui.RenameTreeItem",
+      "dg.jmeter.plugins.templater.RunThroughTree"
+    ],
+    "versions": {
+      "1.1": {
+        "changes": "The yaml config file is used for templates",
+        "downloadUrl": "https://github.com/DanilGolikov/JmeterNamesTemplater/releases/download/v1.1-yaml/jmeter-plugins-names-templater-1.1.jar",
+        "libs": {
+          "snakeyaml": "https://search.maven.org/remotecontent?filepath=org/yaml/snakeyaml/2.2/snakeyaml-2.2.jar"
+        }
+      }
+    }
+  },
+  {
+    "id": "CustomJmeterFunctions",
+    "name": "Custom Jmeter Functions",
```

Comment:
There is existing plugin with this name, you need to choose a different name. Preferably a name that explains what kind of functions are inside. Like "Functions for strings templating" or like that.
Don't forget to change the ID of plugin, and also the description of it should explain more what is inside.